mne/tests/test_epochs.py

```diff
@@ -414,16 +414,13 @@ def test_read_write_epochs():
         # decim with lowpass
         warnings.simplefilter('always')
         epochs_dec = Epochs(raw, events, event_id, tmin, tmax, picks=picks,
-                            baseline=(None, 0), decim=4)
+                            baseline=(None, 0), decim=2)
```

Comment:
Well, I started by searching stuff to optimise, and I found this. Basically this just removes one construction of epochs object while staying essentially the same.